commands/badges.py

```diff
@@ -696,23 +696,62 @@ async def scrap(ctx:discord.ApplicationContext, first_badge:str, second_badge:st
   badge_to_add = db_get_badge_info_by_name(badge_choice)
   badges_to_scrap = [db_get_badge_info_by_name(b) for b in selected_user_badges]
 
-  # Send confirmation message with interaction view
+  # Check for wishlist badges
+  first_scrap_badge = badges_to_scrap[0]
+  first_badge_wishlist_matches = db_get_wishlist_badge_matches(first_scrap_badge['badge_filename'])
+  second_scrap_badge = badges_to_scrap[1]
+  second_badge_wishlist_matches = db_get_wishlist_badge_matches(second_scrap_badge['badge_filename'])
+  third_scrap_badge = badges_to_scrap[2]
+  third_badge_wishlist_matches = db_get_wishlist_badge_matches(third_scrap_badge['badge_filename'])
```

Comment:
I would replace these lines with
```suggestion
  wishlist_matches_groups = [m for m in (db_get_wishlist_badge_matches(b['badge_filename']) for b in badges_to_scrap) if m]
```

That way you don't have to make them into an array later.

Reply:
Very cool! That's good stuff. Thank you!